pages/clusters/manage_clusters.md

```diff
@@ -83,32 +118,107 @@ Modifying the _Allowed IP Addresses_ forcefully disconnects any existing agents
 
 Note the following limitations:
 
-* This setting does not restrict access to the [Metrics API](/docs/apis/agent-api/metrics) for the given cluster token.
-* There is a maximum of 24 CIDR blocks per agent token.
-* IPv6 is currently not supported.
+- This setting does not restrict access to the [Metrics API](/docs/apis/agent-api/metrics) for the given agent token.
+- There is a maximum of 24 CIDR blocks per agent token.
+- IPv6 is currently not supported.
 
-### Migrate to clusters
+## Manage maintainers on a cluster
 
-If you migrate all your existing agents over to clusters, make sure to add all your pipelines to the relevant clusters. Otherwise, any builds for those pipelines will never find agents to run them.
+Buildkite administrators or users with the [_change organization_ permission](/docs/team-management/permissions) can create clusters.
```

Comment:
Where is the 'change organization' permission configurable through?

Doesn't this just mean people who are organization administrator's (who have permission to administer the Buildkite org)?